### DISCUSSION TITLE
reverse search for images with right click

### DISCUSSION THREAD

**Discussion by shalva97** [minbrowser/min#1492] [Ideas]
If i selected text and right lick there is "search with google", this is awesome, but can we have "reverse image search" for images?

Currently I have to download image, then go to some website that can do reverse search and then select that downloaded image. This is all boring. later having my downloads folder filling up with random images is annoying.

**Comment by PalmerAL** [comment]
This is a cool idea!
Probably a good thing to implement as a userscript? You can add a `@run-at context-menu` property to a userscript to get a context menu item for it.
